assets/src/edit-story/components/library/panes/media/local/deleteDialog.js

```diff
@@ -90,13 +94,25 @@ function DeleteDialog({ mediaId, type, onClose }) {
   // Keep icon and menu displayed if menu is open (even if user's mouse leaves the area).
   return (
     <Dialog
-      open
+      isOpen
       onClose={onClose}
       title={type === 'image' ? imageDialogTitle : videoDialogTitle}
       actions={
         <>
-          <Plain onClick={onClose}>{__('Cancel', 'web-stories')}</Plain>
-          <Plain onClick={onDelete}>{__('Delete', 'web-stories')}</Plain>
+          <Button
+            type={BUTTON_TYPES.TERTIARY}
+            size={BUTTON_SIZES.SMALL}
+            onClick={onClose}
+          >
+            {__('Cancel', 'web-stories')}
+          </Button>
+          <Button
+            type={BUTTON_TYPES.PRIMARY}
+            size={BUTTON_SIZES.SMALL}
+            onClick={onDelete}
```

Comment:
Looks like the Dismissal/Cancel button and the primary action button are the same in most of places. I wonder if it would make sense to enable just forwarding the actions directly and adding the buttons by default from the `Dialog`? E.g. `onDismiss`/`onAction` or something similar. Then we wouldn't need to pass the same buttons to the dialogue all the time for the default case with these two buttons. Thoughts?

Reply:
This is a good idea to dry up the code. 

That could work, but then we're tying in the functionality to the design system which is why I kept it this way. If we want to do this above, then I'd say we should perhaps have `edit-story/components/dialog/` where all of this reused stuff everywhere is centralized and then that's used for dialogs so that there's 1 spot controlling the actions in the editor that's built on the design system but the editor's use case is kept inside the editor. 

Would this be a valid compromise?

Reply:
Ah, makes sense, forgot about that part! Good idea to have it as an editor component instead. We could also look into this in a follow-up PR later and keep it as is for now for simplicity.

Reply:
tight! yeah i have to follow this up anyway because the dialogs need to be inverted, that'll actually be super easy to clean up at once. Since this is going to not be merged right away for release purposes i might just swap this back to draft and do it here.

Reply:
ok! updated everything here so we have a shared wrapper for the dialog in the editor (and also the dashboard). And the scrim is fixed. 

Updated dashboard here too because i wanted a reference for what the "inverted" theme would be. Hopefully this doesn't make this too much harder to review and can help shed light.